### ISSUE TITLE
pm.sendRequest does not store cookies in app

### ISSUE THREAD

**Issue by pomeh** [a85/POSTMan-Chrome-Extension#5753]
**Describe the bug**
If I use `pm.sendRequest` to call a service, and this service responds with `Set-Cookie` header(s), those cookies are not stored into Postman UI and not sent with the next "normal request" (without `pm.sendRequest`)

**To Reproduce**
1. Open Postman native App
2. Empty *all* cookies, for *all* domains

3. Open Postman Console

4. Go to Postman main UI, and open a new test tab with, and execute the request:

`GET https://www.google.com` (use whatever domain different than domain below), with following Test script:

```
const req = {
    url: 'https://postman-echo.com/get', # this url SHOULD reply with Set-Cookie header(s)
    method: 'GET',
};

pm.sendRequest(req, function (err, res) {
    console.log('req ended');
});
```

![postman-ui-tab](https://user-images.githubusercontent.com/285534/50910845-ec7a4d00-142e-11e9-86fc-8f3c127a1ea6.png)

5. Go to Postman Console and look for the `https://postman-echo.com/get` call. Response header contains a `Set-Cookie` entry (`Set-Cookie:"sails.sid=s%3AXAb-g48XzYfN7u1[...]; Path=/; HttpOnly"` for me)

![postman-console](https://user-images.githubusercontent.com/285534/50910857-f308c480-142e-11e9-906b-1eb6fa2fc4d3.png)

6. Go to main Postman UI, open Cookies modal

7. No cookie is shown for domain `postman-echo.com`

![postman-ui-cookies](https://user-images.githubusercontent.com/285534/50910861-f603b500-142e-11e9-8e29-6334bdaddf4b.png)

8. Open a new test tab with, and execute the request:

`GET https://postman-echo.com/get` (use same domain as in the previous `pm.sendRequest` call)

9. Go to Postman Console and look for the `https://postman-echo.com/get` call. Request header does not contain any `Cookie` header.

**Expected behavior**
`pm.sendRequest` should (or could ?) share cookies with main Postman UI.
`pm.sendRequest` should/could read and write cookies it receives.

**App information (please complete the following information):**
 - App Type: Postman for Windows / Version 6.6.1 / win32 6.1.7601 / x64
 - Postman Version 6.6.1
 - OS: Windows 7

**Additional context**
If I have cookies for domain D, and I use `pm.sendRequest` to request an url U on this domain D, then the request is sent with cookies. I understand it as: `pm.sendRequest` can *read* cookies from Postman UI, but can not *write* cookies back to it.

I used to test this with Postman version 6.0.10 (with my computer and witch another one on Windows 7 too), and cookies are saved into Postman UI. I've not seen anything in [Postman changelog](https://www.getpostman.com/apps#changelog) that could explains this changes.

I don't know if this behavior is by design, but to me it looks likes a regression, and this is a very nice feature to have. I use it to handle authentication use case (`pm.sendRequest` does the job to authenticate, then I can run my authenticated test).

Maybe, an option could be set to allow or disallow this behavior. A Postman UI global preference, or a per-request preference, something like `pm.sendRequest({ saveCookies: true, ... })`, or `pm.sendRequest({ readCookies: true, writeCookies: true, ... })`.

**Comment by pomeh** [comment]
As a side note, when I do the `pm.sendRequest` call within the Pre-Request script, the cookies are stored ! This could answer to my use case and can be used as a workaround (use `pm.sendRequest` within Pre-Request and not within Request script).

I should have done this test before opening this ticket. But still, I think this behavior could be useful in some cases.

**Comment by codenirvana** [comment]
This is fixed in the latest Postman App (v7.1.0).
